src/main/resources/assets/stylesheets/iva-msva-30.hbs

```diff
@@ -0,0 +1,275 @@
+/* Styling shared by IVA30 and MSVA30 */
+
+/* pdf Page styling */
```

Comment:
1. I think it would be very beneficial to group similar styling here to make this easier to reads.
2. might be better to set the font etc globally instead of setting it individually for p, div, span
3. are these stylings duplicated from the iva30 stylesheet?